### ISSUE TITLE
Hyperlink for mentioned username

### ISSUE THREAD

**Issue by LeGiangAnh** [arkhitech/redmine_mentions#53]
Hi,

I'd like to suggest the feature to add the hyperlink for mentioned username to his profile URL (or people profile URL if use with people plugin (https://www.redminecrm.com/projects/people/pages/1).

Thanks.

**Comment by pyroit** [comment]
agree, in my opinion it would be great if i could be shown on the my page
